drawbicyc/include/drawbicyc/plot_series.hpp

```diff
@@ -501,16 +501,10 @@ void plot_series(Plotter_t plotter, Plots plots, std::string type)
         // Total number of Cloud and Rain Droplets
         try
         {
-          // cloud fraction (cloudy if ql > 1e-5))
-          auto tmp = plotter.h5load_timestep("cloud_rw_mom0", at *n["outfreq"]);
+          auto tmp = plotter.h5load_timestep("cloud_rw_mom0", at *n["outfreq"]) * rhod;
           typename Plotter_t::arr_t snap(tmp);
-          snap *= rhod * plotter.CellVol;
-          auto tmp2 = plotter.h5load_timestep("rain_rw_mom0", at *n["outfreq"]);
-          typename Plotter_t::arr_t snap2(tmp2);
-          snap2 *= rhod * plotter.CellVol;
-          auto tmp3 = plotter.h5load_timestep("all_rw_mom0", at *n["outfreq"]);
-          typename Plotter_t::arr_t snap3(tmp3);
-          res_prof(at) = blitz::sum(snap3);// + blitz::sum(snap2);
+          snap *= plotter.CellVol;
+          res_prof(at) = blitz::sum(snap);
```

Comment:
it would be more efficient to sum all elements first and then multiply the result by cellvol instead of multiplying all elements by the same value and summing afterwards.

Also, currently this only counts cloud droplets. You need to add rain_rw_mom0